### ISSUE TITLE
Support NativeLB option in GatewayAPI provider

### ISSUE THREAD

**Issue by eseiker** [traefik/traefik-helm-chart#1272]
### Welcome!

- [x] Yes, I've searched similar issues on [GitHub](https://github.com/traefik/traefik/issues) and didn't find any.
- [x] Yes, I've searched similar issues on the [Traefik community forum](https://community.traefik.io) and didn't find any.

### What did you expect to see?

After traefik 3.2, `--providers.kubernetesgateway.nativelbbydefault` option is available, but it's not added to this chart.
While kubernetescrd and kubernetesingress providers have the option in this chart, I think it should also be added to gateway provider.

ref: https://github.com/traefik/traefik/pull/11147

**Comment by mloiseleur** [comment]
That's right. In the meantime, it can be added using `additionalArguments`.
